### PR TITLE
DROOLS-2764 : Remove dead code in AuthoringPerspective, AdministrationPerspective and Social Home Page

### PR DESCRIPTION
**Pull request by Rikkola**
https://issues.jboss.org/browse/DROOLS-2764

Bundle:
https://github.com/kiegroup/droolsjbpm-build-bootstrap/pull/783
https://github.com/kiegroup/kie-wb-common/pull/1985
https://github.com/kiegroup/drools-wb/pull/894
https://github.com/kiegroup/optaplanner-wb/pull/292
https://github.com/kiegroup/jbpm-wb/pull/1182
https://github.com/kiegroup/kie-wb-distributions/pull/780
https://github.com/kiegroup/kie-docs/pull/946

### CODE REVIEW

File: drools-wb-webapp/src/main/resources/org/drools/workbench/DroolsWorkbench.gwt.xml [hunk 1]
```diff
@@ -86,7 +86,6 @@
   <inherits name="org.kie.workbench.common.dmn.project.DMNProjectClient"/>
 
   <!-- Social Home page -->
```
Comment (kkufova) [inline drools-wb-webapp/src/main/resources/org/drools/workbench/DroolsWorkbench.gwt.xml]: Can't we remove the whole section?

Comment (Rikkola) [reply drools-wb-webapp/src/main/resources/org/drools/workbench/DroolsWorkbench.gwt.xml]: @kkufova I asked about this and this bit is still needed. I think there is dead code, but some bits are used.

Comment (kkufova) [reply drools-wb-webapp/src/main/resources/org/drools/workbench/DroolsWorkbench.gwt.xml]: @Rikkola, alright!

Comment (kkufova) [reply drools-wb-webapp/src/main/resources/org/drools/workbench/DroolsWorkbench.gwt.xml]: @Rikkola, alright!

**Pull request merged**